lib/rubocop/cli.rb

```diff
@@ -27,7 +27,7 @@ def initialize
     #
     # @param args [Array<String>] command line arguments
     # @return [Integer] UNIX exit code
-    def run(args = ARGV)
+    def run(args = ARGV) # rubocop:disable Metrics/MethodLength
```

Comment:
`Metrics/ClassLength` and `Metrics/MethodLength` each over the maximum length defined in .rubocop_todo.yml.

```consle
% rake spec

(snip)

Offenses:

lib/rubocop/cli.rb:6:3: C: Metrics/ClassLength: Class has too many lines. [188/185]
  class CLI ...
  ^^^^^^^^^
lib/rubocop/cli.rb:30:5: C: Metrics/MethodLength: Method has too many lines. [17/14]
    def run(args = ARGV) ...
    ^^^^^^^^^^^^^^^^^^^^

1081 files inspected, 2 offenses detected
RuboCop failed!
```

I'd make a `# rubocop:disable` comment for the following reasons.

- Raising the number in .rubocop_todo.yml affects the metrix of other codes. It is better not to have as much influence as possible.
- This code is not easy to reduce the number of lines.

Reply:
You forgot to enable the cop after the end of the method.

Reply:
I made a mistake. I fixed it.